### DISCUSSION TITLE
How to find running lsp-servers for a given buffer/workspace by name?

### DISCUSSION THREAD

**Discussion by justinbarclay** [emacs-lsp/lsp-mode#3610] [Q&A] [answered]
Is there a way to find running lsp-servers by name for a workspace? Or, I guess, more specifically: is there a way to see if lsp-eslint is running? For context, I am looking at trying to integrate lsp-eslint into https://github.com/DamienCassou/eslint-disable-rule.

**Comment by yyoncho** [comment, marked as answer]
@justinbarclay there is lsp-find-workspace

**Reply by justinbarclay** [reply]
Thanks a lot :)
